WBooks/BookDetails/BookDetailsViewModel.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  BookDetailsViewModel.swift
+//  WBooks
+//
+//  Created by sebastian.a.spadea on 25/03/2022.
+//
+
+import Foundation
+
+class BookDetailsViewModel {
```

Comment:
Add public to class and protocol. 
Why add protocols?
We use protocols to force the types that make it up to meet the requirements that we want, with variables, methods, etc.